src/Warning.js

```diff
@@ -0,0 +1,14 @@
+class Warning extends Error {
+  constructor (warning) {
+    super()
+
+    const { text, line, column } = warning
+
+    this.name = 'LoaderWarning'
+    this.message = `\n(${line}:${column}) ${text}\n`
```

Comment:
Use `warning.toString()` here  (https://github.com/postcss/postcss/blob/master/lib/warning.es6#L74)

Reply:
I want consistent formatting with the Error Format and the `toString()` message looks weird imho

**Warning**
```
{{ Name }}

({{ Line }}:{{ Column }}) {{ Text }}

```

```
Warning 

(1:1) I'm a Warning

```

**(Syntax)Error**
```
{{ Name }}

({{ Line }}:{{ Column }}) {{ Message }}

{{ Code Frame }}

```

```
SyntaxError

(1:1) I'm an Error

1 |  
2 >|  .class 
3 |

```

Reply:
@michael-ciniawsky :+1: but you should check `line` and `column` anyway, because not all warnings contain this

Reply:
I was not sure about that and took the risk for now, will update

Reply:
@michael-ciniawsky also using own warning text message can lead to loss of information (https://github.com/postcss/postcss/blob/master/lib/warning.es6#L74)

Original
```
postcss-lint:a.css:10:14: Avoid !important
```

You lose `plugin` name, you lose original source map. Be careful postcss `warning` is not error, it is just text message, better use postcss logic for this instead own. There is nothing to worry about that they difference view. Remember - we just wrapper and should provide easy interface. In next version of `postcss` message properties can be changes and it is require modify loader source code, it is not good idea.